build.gradle

```diff
@@ -90,7 +90,7 @@ allprojects {
         testImplementation "org.junit.jupiter:junit-jupiter-api"
         testImplementation "org.junit.jupiter:junit-jupiter-params"
         testImplementation "org.mockito:mockito-junit-jupiter"
-        testImplementation "org.mockito:mockito-core"
+        testImplementation "org.mockito:mockito-inline"
```

Comment:
Out of curiosity. What is mockito-inline and why change from core?

Reply:
To handle static mocks.